src/module/item/entity.js

```diff
@@ -293,33 +307,41 @@ export class OseItem extends Item {
   roll(options = {}) {
     const itemData = this.system;
     switch (this.type) {
-      case "weapon":
+      case "weapon": {
         this.rollWeapon(options);
         break;
-      case "spell":
+      }
+
+      case "spell": {
         this.spendSpell(options);
         break;
-      case "ability":
+      }
+
+      case "ability": {
         if (itemData.roll) {
           this.rollFormula();
         } else {
           this.show();
         }
         break;
+      }
+
       case "item":
-      case "armor":
+      case "armor": {
         this.show();
+      }
     }
   }
 
   /**
    * Show the item to Chat, creating a chat card which contains follow up attack or damage roll options
-   * @return {Promise}
+   *
+   * @returns {Promise}
    */
   async show() {
     const itemType = this.type;
     // Basic template rendering data
-    const token = this.actor.token; //v10: prototypeToken?
+    const { token } = this.actor; // v10: prototypeToken?
```

Comment:
this looks like a note to self that slipped into a commit 😬

Reply:
Hahaha oh no. Do we need to do something with `prototypeToken`?

Reply:
It appears wrong (we'd show the wrong data), so I don't think so?